dev-python/wsproto/wsproto-0.15.0-r1.ebuild

```diff
@@ -18,7 +18,7 @@ IUSE="test"
 RESTRICT="!test? ( test )"
 
 RDEPEND="
```

Comment:
How did you manage to get it to pass tests without listing or packaging correct dependencies?

Reply:
I guess the better question is how does it currently work?

Reply:
Looks like someone already took care of this upstream, maybe they fixed it.

Reply:
It doesn't. But don't worry, I'll take care of packages (now) maintained by python@.